lib/modules/apostrophe-schemas/src/apos/components/ApostropheFieldMixin.js

```diff
@@ -0,0 +1,29 @@
+module.exports = {
```

Comment:
This should live in /src/apos/mixins, not /src/apos/components, use a relative path.